rcl/src/rcl/arguments.c

```diff
@@ -1612,28 +1895,75 @@ _rcl_parse_disabling_flag(
   RCL_CHECK_ARGUMENT_FOR_NULL(suffix, RCL_RET_INVALID_ARGUMENT);
   RCL_CHECK_ARGUMENT_FOR_NULL(disable, RCL_RET_INVALID_ARGUMENT);
 
-  const char * true_prefix = "--enable-";
-  const size_t true_prefix_len = strlen(true_prefix);
+  const size_t enable_prefix_len = strlen(RCL_ENABLE_FLAG_PREFIX);
   if (
-    strncmp(true_prefix, arg, true_prefix_len) == 0 &&
-    strcmp(suffix, arg + true_prefix_len) == 0)
+    strncmp(RCL_ENABLE_FLAG_PREFIX, arg, enable_prefix_len) == 0 &&
```

Comment:
nitpick: While this change is good, it would have been better as a separate commit (for future reference).

Reply:
There was no need to turn these literals into preprocessor variables until I had to reuse them to provide hints in warnings, thus why they are here. Having said, sure, we can split them out.